lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 39px 0 auto;
+    grid-template-rows: 39px auto;
+    gap: 0 16px;
+    grid-template-areas:
+      "logo top-row"
+      "logo bottom-row";
+
+    height: 72px;
+    margin-bottom: 0;
+  }
+
+  .nx-product__top-row {
+    -ms-grid-row-align: start;
+    align-self: start;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    grid-area: top-row;
+    padding-top: $nx-spacing-xs;
+  }
+
+  .nx-product__bottom-row {
+    -ms-grid-row: 2;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    -ms-grid-row-align: start;
+    align-self: start;
+    grid-area: bottom-row;
+    padding-bottom: $nx-spacing-xs;
+  }
+
+  .nx-product__logo {
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: logo;
+  }
+
+  .nx-product__link {
+    display: block;
+    height: 72px;
+  }
+
+  .nx-product__brand {
+    @include bold();
+    color: #2c2c2c;
+    display: inline-block;
+    font-size: 26px;
+    margin: 0;
+  }
+
+  .nx-product__name {
+    color: #2c2c2c;
+    display: inline-block;
+    font-size: 26px;
```

Comment:
Since this and it's sibling are both 26px font-size, that property could be moved to the parent

Reply:
> Since this and it's sibling are both 26px font-size, that property could be moved to the parent

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/fb3ae07c988230b25e10d6b58146a8e517236ec0

Reply:
> > Since this and it's sibling are both 26px font-size, that property could be moved to the parent
>
> fb3ae07

Alright, now it's on the parent and not this element, but it's still also on the sibling which is redundant.

Reply:
> > > Since this and it's sibling are both 26px font-size, that property could be moved to the parent
> > 
> > 
> > [fb3ae07](https://github.com/sonatype/sonatype-react-shared-components/commit/fb3ae07c988230b25e10d6b58146a8e517236ec0)
> 
> Alright, now it's on the parent and not this element, but it's still also on the sibling which is redundant.

The style is coming from the default page header styles.